ecommerce/api.py

```diff
@@ -870,7 +870,11 @@ def fetch_and_serialize_unused_coupons(user):
 
     unused_coupons = []
     for coupon_data in coupons_data:
-        product = Product.objects.get(id=coupon_data["product__id"])
+        try:
+            product = Product.objects.get(id=coupon_data["product__id"])
+        except Product.DoesNotExist:
+            return []
```

Comment:
@tasawernawaz as you worked over that issue before for showing unredeemed coupon codes on dashboard so what you suggest should we return the `[]` or just pass the exception for non existing product ?